gstcefdemux.cc

```diff
@@ -4,7 +4,7 @@
 #include "gstcefaudiometa.h"
 
 #define CEF_VIDEO_CAPS "video/x-raw, format=BGRA, width=[1, 2147483647], height=[1, 2147483647], framerate=[1/1, 60/1]"
-#define CEF_AUDIO_CAPS "audio/x-raw, format=F32BE, rate=[1, 2147483647], channels=[1, 2147483647], layout=interleaved"
+#define CEF_AUDIO_CAPS "audio/x-raw, format=F32LE, rate=[1, 2147483647], channels=[1, 2147483647], layout=interleaved"
```

Comment:
why? :)

Reply:
Ohh no, I wanted to make it a separate thing. I will remove it